### ISSUE TITLE
Broken Next.js SSR after updating v1.4.8 -> v1.4.9.

### ISSUE THREAD

**Issue by jaydenseric** [apollographql/react-apollo#921]
After updating `react-apollo@1.4.8` to the current version, `1.4.9`, `getDataFromTree` imports as `undefined` when attempting SSR in a Next.js project.

Downgrading to `react-apollo@1.4.8` fixes the issue.

**Intended outcome:**

```
import { getDataFromTree } from 'react-apollo'
if (!process.browser) console.log(getDataFromTree)
```

In a Next.js page decorator, this should log the function.

**Actual outcome:**

The import is undefined.

**How to reproduce the issue:**

I have not tested this with the official example code, but my app is not too different: https://github.com/zeit/next.js/blob/v3-beta/examples/with-apollo/lib/withData.js#L3

**Version**
- apollo-client@1.8.1
- react-apollo@1.4.9

**Comment by jbaxleyiii** [comment]
can you `import ReactApollo from "react-apollo"` and tell me what is present?

**Comment by jbaxleyiii** [comment]
@jaydenseric this was a weird UMD bundle issue that I have no idea what caused it! regardless, it is fixed in 1.4.10 (which is released) via https://github.com/apollographql/react-apollo/pull/925
